### ISSUE TITLE
"Something's going on here" message, provided accounts locked out of Discord

### ISSUE THREAD

**Issue by casmint** [nightfall2512/EZ-Discord-Raider#5]
When I try to use this, it joins the server, but doesn't send messages. It also won't let them leave using the massleave feature. When I tried joining a new server with the accounts, it gave me the message `{"message": "You need to verify your account in order to perform this action.", "code": 40002}`. Then when I tried logging into one of the accounts manually, it says "Something's going on here" and I need to verify the account with a phone number. Discord seems to detect the accounts being controlled automatically and then locks them out. Is there a possible fix for this?

**Comment by cateee69** [comment]
Dont use joiner then

**Comment by casmint** [comment]
> Dont use joiner then

yeah but it's an issue with the program, you can't solve it by saying "oh just don't use it"

**Comment by cateee69** [comment]
i fixed joiner tho you can find it in pull requests tab

**Comment by casmint** [comment]
cool ok, you didn't specify that. my bad
